connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/IncrementalCooperativeAssignor.java

```diff
@@ -463,6 +464,14 @@ protected void handleLostAssignments(ConnectorsAndTasks lostAssignments,
             lostAssignmentsToReassign.addConnectors(lostAssignments.connectors());
             lostAssignmentsToReassign.addTasks(lostAssignments.tasks());
             return;
+        } else if (maxDelay == 0) {
+            log.debug("Scheduled rebalance delays are disabled ({} = 0); "
+                    + "reassigning all lost connectors and tasks immediately",
+                    SCHEDULED_REBALANCE_MAX_DELAY_MS_CONFIG
+            );
+            lostAssignmentsToReassign.addConnectors(lostAssignments.connectors());
+            lostAssignmentsToReassign.addTasks(lostAssignments.tasks());
```

Comment:
nit: These 2 lines along with https://github.com/apache/kafka/blob/trunk/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/IncrementalCooperativeAssignor.java#L463-L464 and https://github.com/apache/kafka/blob/trunk/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/IncrementalCooperativeAssignor.java#L501-L502 are duplicates. Can we create a separate method  and move this logic there?

Reply:
Ack, done 👍

Reply:
Nice 👍